frontend/src/pages/analytics/dashboard/detail/AnalyticsDashboardDetail.tsx

```diff
@@ -26,7 +26,7 @@ const AnalyticsDashboardDetail: React.FC = () => {
       const { success, data }: ApiResponse<any> = await fetchEmbeddingUrl(
```

Comment:
Do we still have to use mock data here?

Reply:
Yes, wait dashboard api ready in other PR.